### ISSUE TITLE
guide: extract `remote add/modify` (advanced) config details to UG

### ISSUE THREAD

**Issue by skshetry** [iterative/dvc.org#1859]
UPDATE: Jump to https://github.com/iterative/dvc.org/issues/1859#issuecomment-708084238

UPDATE 2: Some related tickets that could potentially be included in this refactor:
- [ ] https://github.com/iterative/dvc.org/issues/1699
- [ ] https://github.com/iterative/dvc.org/issues/1762

---

It'd be great if we could share the link to that container directly (that on opening would expand the container and scroll that touser's viewport).

eg: The following section on [command-reference/run#description](https://dvc.org/doc/command-reference/run#description).
![Capture](https://user-images.githubusercontent.com/18718008/95848248-a1c3ee00-0d02-11eb-99fb-afdbf0e9d507.PNG)

**Comment by jorgeorpinel** [comment]
Would be nice to have. Probably not very hard to implement either but I'm not convinced there's a huge need. We indeed have already a problem of linking to anchors too much already — it's best to avoid that as headers tend to change which causes semi-broken links which are very hard to detect.

**Comment by shcheklein** [comment]
My take on this- if it happens with a certain section then it probably means that the section should not be collapsible- rather regular section, like the one @skshetry mentioned. In a lot of cases we use collapsible sections as a substitute for the whole page (e.g. remotes).

**Comment by jorgeorpinel** [comment]
Good point. So should we change this to reviewing the existing `<details>` sections? E.g. are you saying we should use regular H3 sections in remote add and modify @shcheklein ?

**Comment by shcheklein** [comment]
>   So should we change this to reviewing the existing <details> sections?

let's do this as we go, care by case

> E.g. are you saying we should use regular H3 sections in remote add and modify @shcheklein ?

yes, if there is a feeling that we want to share a link, or that information is important enough, not auxiliary

**Comment by jorgeorpinel** [comment]
> do this as we go

Got it. So, close this issue (as not actionable)

> that information is important enough, not auxiliary

In the case of remote add/modify examples the usage of the details sections is not to include auxiliary info but simply to make the docs more compact. They would be really really long otherwise. The other approach would be to break up the doc into several pages but it's not clear to me how (without new engine features).

**Comment by shcheklein** [comment]
> Got it. So, close this issue (as not actionable)

why? we can make specifically this section H3?

> The other approach would be to break up the doc into several pages but it's not clear to me how (without new engine features).

just break like we do for Google Drive (to some extent)? Just create separate pages and link them from those small collapsibles ?

**Comment by jorgeorpinel** [comment]
> can make specifically this section H3

I think that was just a generic example. Or are we trying to actually link to **Avoiding unexpected behavior** in command-reference/run#description. from somewhere @skshetry ?

> Just create separate pages

Unlisted pages? (not in nav) Yeah that could work, it's just not something we do much currently.

**Comment by shcheklein** [comment]
> Unlisted pages? (not in nav) Yeah that could work, it's just not something we do much currently.

no, I mean regular page. We should not be doing unlisted pages (why?). Similar to GDrive.

**Comment by jorgeorpinel** [comment]
> break like we do for Google Drive
> I mean regular page

OK so a single guide specifically for remote add/modify configuration... OK, that should work, repurposing issue!

**Comment by shcheklein** [comment]
> OK, that should work, repurposing issue!

I think, let's wait on @skshetry .. it might be he wanted to share this specific link

**Comment by jorgeorpinel** [comment]
A good first attempt (specific to Azure): https://github.com/iterative/dvc.org/pull/2634

**Comment by jorgeorpinel** [comment]
Is this something you'd be interested in beginning @casperdcl ? Still needed much. Thanks

**Comment by daavoo** [comment]
> A good first attempt (specific to Azure): #2634

Another user has created something that could be part of a `Setup a Microsoft Azure DVC Remote` guide:

https://github.com/iterative/dvc/issues/2200#issuecomment-921030176

**Comment by shcheklein** [comment]
I think we have prematurely renamed this issue. This was not about remotes, it was about `run`?

**Comment by jorgeorpinel** [comment]
Hmm yeah I probably didn't notice your message in https://github.com/iterative/dvc.org/issues/1859#issuecomment-708085820 and then @skshetry never followed up so this issue was repurposed. I believe he meant this as a general feature though, either for that `run` ref example or other places... I guess I should reopen both that issue and the one about moving remote ref details to the UG ⏳

**Comment by jorgeorpinel** [comment]
See https://github.com/iterative/dvc.org/issues/2865

and https://github.com/iterative/dvc.org/issues/2866
